packages/runtime/lib/config.js

```diff
@@ -87,8 +87,9 @@ async function _transformConfig (configManager, args) {
   for (let i = 0; i < services.length; ++i) {
     const service = services[i]
 
-    // We need to have absolut paths here, ot the `loadConfig` will fail
-    if (!isAbsolute(service.path)) {
+    // We need to have absolute paths here, ot the `loadConfig` will fail
+    // Make sure we don't resolve if env var was not replaced
+    if (service.path && !isAbsolute(service.path) && !service.path.match(/^\{.*\}$/)) {
```

Comment:
Why is this needed? In theory, it shouldn't be.

Reply:
It is not for the default case.
But i found a regression if `replaceEnv=false` in the `ConfigManager` which lead to to paths like `/this/is/root/{PLT_ENV}` or `/this/is/root` when `path=''`.